### ISSUE TITLE
Changing path URL

### ISSUE THREAD

**Issue by blueberry08** [integry/livecart#65]
Hi, I want to move the store from /home/user/public_html/store to /home/user/public_html/shop but I always get the following error when I change the folder name:

Fatal error: Uncaught exception 'ClassLoaderException' with message 'File /home/user/public_html/store/application/LiveCart.php not found' in /home/user/public_html/shop/framework/ClassLoader.php:128 Stack trace: #0 /home/user/public_html/shop/framework/ClassLoader.php(87): ClassLoader::load('/home/user/...') #1 [internal function]: ClassLoader::load('LiveCart') #2 /home/user/public_html/shop/public/index.php(47): spl_autoload_call('LiveCart') #3 {main} thrown in /home/user/public_html/shop/framework/ClassLoader.php on line 128

How should I fix this? Thanks.

**Comment by rokoso** [comment]
Did you try to make a new folder 'shop' and then copy all contents from folder 'store' in 'shop'. Then change the url in file \storage\configuration\url.php ?

**Comment by blueberry08** [comment]
Yes, that actually worked. Now, how do I delete the store folder safely so I can retain only the shop folder? Because every time I delete it, the same error shows up again.

**Comment by rokoso** [comment]
You did change the url in the \storage\configuration\url.php of 'shop' folder (not 'store' folder), right ?

**Comment by blueberry08** [comment]
Yes, everything remains unchanged in the store folder. I just copied it then edited url.php in the shop folder. I don't know why it's referring to /home/user/public_html/store/application/LiveCart.php in the error when I delete it.

**Comment by rokoso** [comment]
Try to delete the contents of the /cache and /public/cache folders.

**Comment by blueberry08** [comment]
Deleted everything in the /cache and /public/cache folders, still didn't work. Don't you think it has something to do with the Apache mounting path?
